core/src/components/button/test/theme-ionic/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
```

Comment:
You can add `theme="ionic"` to the `html` node to avoid having to specify it on any children elements.

Reply:
I used the theme="ionic" at the ion-button directly since we do not have ionic theme styles for the header yet. This way screen will have a better appearance. That said, I would say once we get everything styled we can remove this inline theme attrs ;)